### ISSUE TITLE
Automatically establish some mirrors when uploading a file using the CLI

### ISSUE THREAD

**Issue by bookchin** [Storj/core#332]

**Comment by lakewik** [comment]
I tried several times excute create-mirrors but i'm getting 
`502 Bad Gateway` error response from server. This happen almost always with files larger than 5MB.

**Comment by super3** [comment]
@lakewik Please make an issue, and include as much detail as possible.

**Comment by bookchin** [comment]
Let's hold off on this one temporarily until we improve the bridge handling for mirror establishment.

**Comment by bookchin** [comment]
https://github.com/Storj/bridge/issues/195

**Comment by bookchin** [comment]
closing in favor of having the bridge do this automatically after #415 is completed
